packages/client/src/components/ui/pagination.vue

```diff
@@ -62,24 +74,72 @@ const props = withDefaults(defineProps<{
 });
 
 const emit = defineEmits<{
-	(e: 'queue', count: number): void;
+	(ev: 'queue', count: number): void;
 }>();
 
-type Item = { id: string; [another: string]: unknown; };
+let rootEl = $ref<HTMLElement>();
 
-const rootEl = ref<HTMLElement>();
-const items = ref<Item[]>([]);
-const queue = ref<Item[]>([]);
+/*
+ * 遡り中かどうか
+ */
+let backed = $ref(false);
+
+let scrollRemove: (() => void) | null = $ref(null);
```

Comment:
I think since `ref` is used below, this should not use `$ref`. The type definition is in the wrong place.
```suggestion
let scrollRemove = ref<(() => void) | null>(null);
```